stack/corepersistence/queue/src/main/java/org/apache/usergrid/persistence/queue/settigs/IndexConsistency.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
.../settigs/IndexConsistency -- this is a duplicate file

Reply:
removed